<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { background-color: white; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>201</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget { background-color: lightgrey; }</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>292</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="reset_original">
        <property name="styleSheet">
         <string notr="true">background-color: lightblue;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Revert to Original</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="avg_filter">
        <property name="styleSheet">
         <string notr="true">background-color: white;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Average Filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="salt_pepper">
        <property name="styleSheet">
         <string notr="true">background-color: white;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Salt and Pepper Noise</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mdn_filter">
        <property name="styleSheet">
         <string notr="true">background-color: white;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Median Filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hpass_filter">
        <property name="styleSheet">
         <string notr="true">background-color: white;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Highpass Filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unsharp_filter">
        <property name="styleSheet">
         <string notr="true">background-color: white;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Unsharp Filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hboost_filter">
        <property name="styleSheet">
         <string notr="true">background-color: white;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Highboost Filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sobel_gradient">
        <property name="styleSheet">
         <string notr="true">background-color: white;
border-radius: 5px;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Sobel Gradient</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>571</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="orig_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Original Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="origimg">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>277</width>
           <height>225</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid black</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="img2_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="img2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>278</width>
           <height>225</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid black</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="img3_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="img3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>278</width>
           <height>225</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid black</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="img1_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="img1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>278</width>
           <height>225</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid black</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
